docs/private-dns/connect-devices/mobile-and-desktop/linux.md

```diff
@@ -102,6 +102,21 @@ If you see a notification that you are not connected to AdGuard DNS, most likely
 
 :::
 
+## Use EDNS (Extended DNS)
+
+EDNS extends the DNS protocol, enabling larger UDP packets to carry additional data. In AdGuard DNS, it allows passing the DeviceID in plain DNS using an extra parameter.
+
+DeviceID, an eight-digit hexadecimal identifier (e.g., `1a2b3c4d`), helps link DNS requests to specific devices. For encrypted DNS, this ID is part of the domain (e.g., `1a2b3c4d.d.adguard-dns.com`). For unencrypted DNS, EDNS is required to localize this identifier.
+
+AdGuard DNS uses EDNS to retrieve the DeviceID. All you have to do is look for option number 65074 and, if such an option exists, read the DeviceID from there. For this, we need to use the command `dig` on the terminal:
```

Comment:
The second sentence has a weird subject, since it is actually AdGuardDNS that is supposed to look for the option, not the users.

The third sentence is slightly incorrect, since the `dig` command is merely an example, and one can actually use any DNS software with an ability to add EDNS options.